wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+
+```markdown
+osu! is installed in the `AppData/Local` folder by default, unless specified otherwise during installation.
+```
+
+### Keywords and commands
+
+When copying a keyword or command, the letter casing should be copied as it appears or how someone normally would type it. If applicable, prefer lowercase letters. (Example shown below.)
+
+```markdown
+As of now, the `Name` and `Author` commands in the skin configuration file (`skin.ini`) do nothing.
+```
+
+### File names
+
+When copying the name of a file, the letter casing should be copied as it appears. If applicable, prefer lowercase letters. (Example shown below.)
+
+```markdown
+To play osu!, double click the `osu!.exe` icon.
+```
+
+### File extensions
+
+*Notice: File formats (not to be confused with file extensions) must be written in capital letters without the prefixed fullstop (`.`).*
+
+File extensions must be prefixed with a fullstop (`.`) and be followed by the file extension in lowercase letters. (Example shown below.)
+
+```markdown
+The JPG (or JPEG) file format has the `.jpg` (or `.jpeg`) extension.
+```
+
+### Chat channels
+
+When copying the name of a chat channel, start it with a hash (`#`), followed by the channel name in lowercase letters. (Example shown below.)
+
+```markdown
+`#lobby` is where you can advertise your multi room.
+```
+
+## Preformatted text (code blocks)
+
+*Notice: Syntax highlighting for preformatted text is not implemented on the website yet.*
+
+Preformatted text (also known as code blocks) must be fenced using three grave marks. They should set the language identifier for syntax highlighting.
+
+## Links
+
+There are two types of links: inline and reference. Inline has two styles.
+
+The following is an example of both inline styles:
+
+```markdown
+[Game Modifiers](/wiki/Game_Modifiers)
+
+<https://osu.ppy.sh/home>
+```
+
+The following is an example of the reference style:
+
+```markdown
+[Game Modifiers][game mods link]
+
+[game mods link]: /wiki/Game_Modifiers
+```
+
+---
+
+Links must use the inline style if they are only referenced once. The inline angle brackets style should be avoided. References to reference links must be placed at the bottom of the article.
+
+### Internal links
+
+*Note: Internal links refer to links that stay inside the `https://osu.ppy.sh/` domain.*
+
+#### Wiki links
+
+All links that point to an wiki article should start with `/wiki/` followed by the path to get to the article you are targeting. Relative links may also be used. Some examples include the following:
+
+```markdown
+[FAQ](/wiki/FAQ)
+[pippi](/wiki/Mascots#-pippi)
+```
+
+---
+
+Wiki links must not use redirects.
+
+---
+
+Wiki links must not use a trailing forward slash (`/`).
```

Comment:
I think there's no need in separating these and you can just merge them into one small paragraph and add a good/bad example for a redirect or its absence (may use `ASC` as an example, even), otherwise the section looks a bit bloated